iotilecore/RELEASE.md

```diff
@@ -2,6 +2,14 @@
 
 All major changes in each released version of `iotile-core` are listed here.
 
+## 4.1.3
```

Comment:
Your version.py only lists 4.1.2 

Depending on what you're intending to do, there should only be one version bump in the RELEASE.md per PR, and if you intend to release the changes should probably be bundled together in 4.1.2 since it hasn't been cut yet.

Reply:
Fixed.